starter-repos/app-template/config/envs/prod-central/kustomization.yaml

```diff
@@ -15,7 +15,7 @@
 # Customize the namespace to be the namespace name used for prod-central environment.
 # This change should be made before `appcel env add prod-central`.
 # Once the prod-central environment is added, the namespace shouldn't be changed.
-namespace: prod-central
+namespace: go-app
```

Comment:
So the same `go-app` namespace will be used to deploy the app in staging/prod clusters?

Reply:
Yes, that is the namespace I put in the root repo.